modules/platform/nuxeo-platform-document-routing/nuxeo-routing-core/src/main/java/org/nuxeo/ecm/platform/routing/core/impl/DocumentRoutingServiceImpl.java

```diff
@@ -1388,4 +1392,52 @@ public void invalidateRouteModelsCache() {
     public boolean isWorkflowModel(final DocumentRoute documentRoute) {
         return documentRoute.isValidated();
     }
+
+    /**
+     * @since 11.3
+     */
+    @Override
```

Comment:
No need for Javadoc on the implementation, it's already on the interface.

Reply:
done